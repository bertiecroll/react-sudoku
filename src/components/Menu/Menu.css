.Menu {
  margin: 1em;
  padding: 1em;
  display: flex;
  flex-direction: column;
  align-items: center;
  color: #999;
}

.Menu h2 {
  margin-bottom: .1em;
}

.Menu p {
  margin-top: .1em;
}

.Menu a {
  width: 12em;
  margin: .5em;
  padding: .75em;
  text-align: center;
  border-radius: .5em;
  text-decoration: none;
  background-color: #67DAF9;
  box-shadow: 1px 1px 1px 0px #000;
  color: #fff;
}
