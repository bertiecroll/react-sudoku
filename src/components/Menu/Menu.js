import React from 'react'
import { Link } from 'react-router-dom'

import './Menu.css';

export default function Menu (props) {
  return (
    <div className="Menu">
      <h2>Welcome</h2>
      <p>Choose your difficulty level to begin.</p>
      <Link to="/puzzle/easy">Just Getting Started</Link>
      <Link to="/puzzle/medium">Middle Of The Road</Link>
      <Link to="/puzzle/hard">Ready For A Challenge</Link>
    </div>
  )
}
