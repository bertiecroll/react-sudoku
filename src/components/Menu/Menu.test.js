import React from 'react';
import { shallow } from 'enzyme'
import Menu from './Menu';

describe('Menu', () => {
  it('renders component content', () => {
    const wrapper = shallow(<Menu />)
    expect(wrapper).toMatchSnapshot()
  })
})
