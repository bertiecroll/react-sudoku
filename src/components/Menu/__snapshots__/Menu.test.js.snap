// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders component content 1`] = `
<div
  className="Menu"
>
  <h2>
    Welcome
  </h2>
  <p>
    Choose your difficulty level to begin.
  </p>
  <Link
    replace={false}
    to="/puzzle/easy"
  >
    Just Getting Started
  </Link>
  <Link
    replace={false}
    to="/puzzle/medium"
  >
    Middle Of The Road
  </Link>
  <Link
    replace={false}
    to="/puzzle/hard"
  >
    Ready For A Challenge
  </Link>
</div>
`;
